x/records/module_ibc.go

```diff
@@ -187,8 +187,7 @@ func (im IBCModule) OnAcknowledgementPacket(
 	if data.Sender == im.keeper.AccountKeeper.GetModuleAddress(stakeibctypes.ModuleName).String() {
 		switch resp := ack.Response.(type) {
 		case *channeltypes.Acknowledgement_Result:
-			log := fmt.Sprintf("\t [IBC-TRANSFER] Acknowledgement_Result {%s}", string(resp.Result))
-			im.keeper.Logger(ctx).Info(log)
+			im.keeper.Logger(ctx).Info(fmt.Sprintf("\t [IBC-TRANSFER] Acknowledgement_Result {%s}", string(resp.Result)))
```

Comment:
I think this prints gibberish
![Screen Shot 2022-08-13 at 5 33 11 PM](https://user-images.githubusercontent.com/19301583/184515749-7146b813-adfc-4223-9fc7-df2a89345828.png)

Reply:
it just prints the bytes from the ack (possibly useful for debugging)